src/Dashboard/Data/FunctionInstanceSnapshot.cs

```diff
@@ -17,7 +17,7 @@ public class FunctionInstanceSnapshot
 
         public Guid Id { get; set; }
 
-        public Guid HostInstanceId { get; set; }
```

Comment:
Oh, shoot, but the dashboard is deployed at a different cadence. So it's possible that we are writing logs with strings yet the Dashboard reads them as Guids and fails.